src/main/resources/openapi/audio.yaml

```diff
@@ -226,6 +226,32 @@ paths:
                 type: "AUDIO_101"
                 title: "The requested data cannot be located"
                 status: 500
+  /audios:
+    post:
+      tags:
+        - audio
+      summary: Upload a audio meta data
```

Comment:
Typo: Maybe this summary should be `Upload audio metadata`?

Reply:
ACCEPT